pdfminer/utils.py

```diff
@@ -17,12 +19,16 @@ class open_filename(object):
     (just like `open`), but does nothing for file-like objects.
```

Comment:
Also add `pathlib.PurePath` to the docstring.

Reply:
Addressed in db64da2, but please correct it if needed as I'm not a native English speaker.